### ISSUE TITLE
Chatrooms

### ISSUE THREAD

**Issue by nukeop** [rms-support-letter/rms-support-letter.github.io#1863]
Discord: https://discord.gg/7FWkxG4CsU

Matrix.org: #free-rms:matrix.org

**Comment by nukeop** [comment]
Yeah I know - non-free

**Comment by shenlebantongying** [comment]
Cannot we just chat here? Its perfect enough somehow.

**Comment by nukeop** [comment]
We can - this is completely optional. Discord is also available via web interface.

**Comment by clort81** [comment]
Why would you use censor-heavy chat platforms?  There's talko.cc, there's rizon IRC.

**Comment by nukeop** [comment]
100-1000x more people use Discord. That's the reality. I've never seen anything censored on Discord anyway, and I don't see why they'd seek to do anything to a small chatroom of this letter's supporters.
If you want to create chatrooms there, please by all means do, and I'll add links. Also looking for someone to maintain a Matrix.org room...

**Comment by clort81** [comment]
Don't use it.  That's the reality.  It's evil.  That's the reality.

**Comment by nukeop** [comment]
Lots of people were also saying to not use Github (I don't think even RMS himself is that extreme). If we did that there would be no letter, no community, no support, and no opposition to the open-letter.

**Comment by brown121407** [comment]
@clort81 if you know how to create and manage a channel on IRC, please do. I'm sure a lot of us would love to join (me included), and we can link that besides the Discord and Matrix.

**Comment by shenlebantongying** [comment]
There is an official `#fsf ` channel, we don't have to create a new one. https://www.gnu.org/server/irc-rules.en.html

**Comment by nukeop** [comment]
Isn't it controlled by the non-board members opposed to RMS?

**Comment by shenlebantongying** [comment]
I suppose they are modded by fsf's member or staffs/

**Comment by brown121407** [comment]
Some are now talking in favor of RMS right now on `#fsf` so it might not be a bad idea to point people there. At least maybe we can turn the tide there in the positive direction if some oppose. (Assuming we behave ourselves.)

**Comment by m-ar-c** [comment]
I have an xmpp server, do you want me to create a jabber public chatroom ?

**Comment by shenlebantongying** [comment]
I do think we should remove discord on the README. Also Discrod has been bought by Microsoft recently.

**Comment by nukeop** [comment]
If we remove it no one will join. It's completely optional, if you feel it's not for you, just don't join it.

**Comment by nukeop** [comment]
> I have an xmpp server, do you want me to create a jabber public chatroom ?

Sure

**Comment by imachug** [comment]
I've moved Discord to the bottom of the list and added IRC and matrix.org. I'm closing this issue now because there are too many open issues already and everything is now on README. Please create a PR if you create a new room somewhere (e.g. XMPP, would be awesome).

**Comment by m-ar-c** [comment]
Ok, here it is: xmpp:support-rms@conf.marc-o.win?join . Before making it public it would be useful to test it (I only made private room for friends and family already on my server so far).

**Comment by nukeop** [comment]
Let's keep it opened and pinned so more people know where to go to chat

**Comment by imachug** [comment]
> Let's keep it opened and pinned so more people know where to go to chat

We already have the list on README, no?

**Comment by nukeop** [comment]
Yeah, but some redundancy is needed as people land in the repo in different subpages. I'd like to keep the most important links from organizational point of view pinned here.

**Comment by venomega** [comment]
what about using bridges between the rooms?

**Comment by imachug** [comment]
It'd be nice, but all bridges I know have ~ 5 minute delay which is very bad for active conversations. Do you know any fast bridge?

**Comment by RealJTG** [comment]
@imachug check https://github.com/42wim/matterbridge - "bridge between mattermost, IRC, gitter, xmpp, slack, discord, telegram, rocketchat, twitch, ssh-chat, zulip, whatsapp, keybase, matrix, microsoft teams, nextcloud, mumble, vk and more with REST API"

**Comment by venomega** [comment]
i tested matterbridge before, it works great

**Comment by shenlebantongying** [comment]
No, we shouldn't use/support any proprietary platforms. That's against what we are doing.  IRC is good, gitter is ok. 

But not discord or slack. That horrible things for us.

**Comment by M-i-k-o-t-o** [comment]
@m-ar-c 

> Ok, here it is: xmpp:[support-rms@conf.marc-o.win](mailto:support-rms@conf.marc-o.win)?join . Before making it public it would be useful to test it (I only made private room for friends and family already on my server so far).

"remote server not found"

@shenlebantongying
I agree with you, although the sad part is that github itself is a nonfree platform.

**Comment by nukeop** [comment]
By definition, platforms aren't free or non-free, only their clients can be (server software is typically not distributed).

But I am ok with using proprietary software if it furthers my goals. It's a tradeoff, and not all proprietary software limits your freedom in the same way. Some limit it in ways that are insignificant compared to the benefits you gain from them, especially short term. I know it's all heresy but it's an unassailable fact that without the "proprietary" platform of Github we would have 10-20 signatures and not 2000.

**Comment by jobbautista9** [comment]
The FSF uses Twitter (which is proprietary), if I remember correctly. GNU also writes free software in proprietary operating systems. It's a necessary evil.

If you're concerned about Discord, you can always use a discord-to-irc proxy (like bitlbee's discord gateway). Or just not use it at all and stick with IRC.

There are also people who are not 100% supportive/purist of free software, but are concerned about the FLOSS being attacked by the woke pseudo-leftist mob. They might be using Discord. We shouldn't alienate those allies.

**Comment by xTpx** [comment]
Why support GNU Emacs on Windows?

We hope that the experience of using GNU Emacs on Windows will give
programmers a taste of freedom, and that this will later inspire them
to move to a free operating system such as GNU/Linux. That is the main
valid reason to support free applications on nonfree operating
systems.
https://www.gnu.org/software/emacs/manual/html_mono/efaq-w32.html#Why-Emacs-on-Windows

**Comment by crappyrules** [comment]
> I do think we should remove discord on the README. Also Discrod has been bought by Microsoft recently.

github was bought by microsoft too brainlet

**Comment by kompowiec** [comment]
> Yeah I know - non-free

![absolutely proprietary](https://pbs.twimg.com/media/ECkcrrWUIAEepaO.jpg)

**Comment by Aspie96** [comment]
Why discord and not gitter?

**Comment by nukeop** [comment]
We already have 6 different channels, which is already almost too much for me to handle - I don't really want to add any more.

**Comment by m-ar-c** [comment]
@M-i-k-o-t-o 

> @m-ar-c
> 
> "remote server not found"
> 

Sorry I just saw your message (way too much notifications these days :o). It should work now. Tell me if it doesn't.

**Comment by andrew-phi** [comment]
@m-ar-c server does not exist.

**Comment by m-ar-c** [comment]
@M-i-k-o-t-o @andrew-phi  Ok, now it should really work ! DNS issue.

**Comment by m-ar-c** [comment]
5 people on the jabber room support-rms@conf.marc-o.win . Working well. You can make it public if you want.

**Comment by venomega** [comment]
> 5 people on the jabber room [support-rms@conf.marc-o.win](mailto:support-rms@conf.marc-o.win) . Working well. You can make it public if you want.

how can i join in? i`m using profanity, i dont have a user there either

**Comment by m-ar-c** [comment]
@venomega I don't know or use profanity, but the documentation ( https://profanity-im.github.io/guide/0100/rooms.html#joining-a-chat-room ) says you need to do `/join room@conference.server.com` . In this case it would be `/join support-rms@conf.marc-o.win`

**Comment by cmpunches** [comment]
Hey can we remove the Libera mentions?

That shift in staff is not being presented correctly and consists mostly of toxic admins that were driven out by new ownership at freenode after decades of them abusing people on their network -- I don't think you want to associate that effort with this project because in less than 48 hours it has basically become a toxic admin paradise.

**Comment by nukeop** [comment]
It's just a chatroom, I don't know what you're talking about.
